cpu/cortexm_common/include/cpu.h

```diff
@@ -36,6 +36,7 @@
 #include "sched.h"
 #include "thread.h"
 #include "cpu_conf.h"
+#include "bitband.h"
```

Comment:
Convenience or lazy, this makes the big macros immediately available in all files using cpu.h